src/core_plugins/kibana/public/visualize/editor/editor.js

```diff
@@ -115,6 +127,8 @@ function VisEditor($scope, $route, timefilter, AppState, $location, kbnUrl, $tim
     docTitle.change(savedVis.title);
   }
 
+  // Extract visualization state with filtered aggs.
+  // Consists of: aggs, params, listeners, title, and type.
```

Comment:
I think this should probably point the reader to where they can find the current state (like the URL) rather than list out it's properties. Otherwise the list will just get out of date and not benefit anyone